stl/inc/type_traits

```diff
@@ -331,6 +331,28 @@ _INLINE_VAR constexpr bool is_array_v<_Ty[]> = true;
 template <class _Ty>
 struct is_array : bool_constant<is_array_v<_Ty>> {};
 
+#if _HAS_CXX20
+// STRUCT TEMPLATE is_bounded_array
+template <class>
+_INLINE_VAR constexpr bool is_bounded_array_v = false;

```

Comment:
Since this is C++20-only code, it need not use the `_INLINE_VAR` macro - which expands to nothing in sub-C++17 mode and `inline` in C++17 or greater modes - and can simply use `inline` directly.

Reply:
True, my thought was that `_INLINE_VAR` is not really that much longer than `inline` but is is more consistent with the code around. 

Will fix anyway